### DISCUSSION TITLE
To show 4 fields in a row desktop and 2 fields in a row in mobile

### DISCUSSION THREAD

**Discussion by phpsmartteam** [ColorlibHQ/AdminLTE#5624] [General]
Hi,

We are showing 4 fields like Date of Birth, Age, Gender and Active in one row in desktop

If we go for responsive in mobile view these are coming in 4 rows (one field per row). How can we show in two rows like below
Row 1: Date of Birth, Age
Row 2: Gender, Active

Please suggest

Thanks

**Comment by danny007in** [comment]
is this issue still exist ?

**Reply by sierratec** [reply]
No, we have resolved this by applying a custom class.

Thanks!
